nafiesl/uptime-monitor
# Check Now Button

### Description

In this PR, we are adding a new Check Now button on the customer site detail page. When we click the button, it will run the check on the customer's website.

### Task Link

- #1 

### Checklist

- [x] Check Now button
- [x] Refactor check run into a RunCheck job class
- [x] Add queue table for database queue driver

### How to test

1. Open one customer site detail page
2. Click the Monitoring Log tab
3. Click the Check Now button
4. Verify that a new log has been added and the Last check at value is update (x seconds ago).

### Screenshot

The check now button on the customer site detail page

![screen_2024-01-04_010](https://github.com/nafiesl/uptime-monitor/assets/8721551/6c1fbfc7-e50e-452d-baf2-c00adcff42e8)
